priority-queue/PriorityQueueConsumerHigh/local.settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+  },
+  "ConnectionStrings": {
+    "ServiceBusConnectionString": "Endpoint=sb://[your namespace].servicebus.windows.net;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=[your secret]"
```

Comment:
When going to 5.x version of the extension, you should be able to move this to managed identity.  No SaS keys going forward :)  https://docs.microsoft.com/azure/azure-functions/functions-bindings-service-bus-trigger?tabs=csharp#identity-based-connections

Reply:
Done ! everything is much cleaner now

Reply:
So much cleaner now, thanks.